### PR TITLE
Fix responsive issues

### PR DESCRIPTION
**Pull request by Pallavi-PH**
Signed-off-by: Pallavi-PH <pallaviph02@gmail.com>

This PR fixes the following issues:

1. Fix issues on blog screen

- Font size of title and description in mobile screen
- Text color of blog description in mobile screen
- Tag button border-radius
- Tag button font-size in mobile screen
- Author avatar size
- Author text color and spacing, font-size
- Time font-size
Screenshot 2021-06-21 at 5 09 20 PM

2. Add white background for mobile header
3. Fix page header issues on the author page

- Adjust font-size of Author and author description
- Author avatar and name alignment
- Add separate background image to mobile screen
- Adjust spacing on mobile screen header
Screenshot 2021-06-21 at 5 48 08 PM

4. Blog cards alignment on the author page
Screenshot 2021-06-21 at 5 53 28 PM

5. Hide 'Read more' button on mobile screen
6. Fixes on the commercial support page

- Fix image overflow in the commercial support page for mobile screen
Screenshot 2021-06-21 at 5 56 13 PM

- Fix logo stretching by keeping only height fixed and letting width adjusting auto
- Adjust around spacing for cards in mobile screen
Screenshot 2021-06-21 at 5 57 09 PM

7. Fix issues on the newsletter

- Removed box-shadow for 'Subscribe' button
- Adjust spacing for mobile and desktop
Screenshot 2021-06-21 at 5 59 59 PM

8. Fix testimonial width and button overflow issue for mobile screen on home screen
Screenshot 2021-06-21 at 6 12 12 PM

### CODE REVIEW

**Review by satyapriyamishra222 — APPROVED** [review]

File: website/src/components/DisplayAuthorandReadTime/styles.ts [hunk 1]
```diff
@@ -18,13 +22,18 @@ const useStyles = makeStyles((theme: Theme) => ({
...
+    fontSize: '1rem',
+    padding: theme.spacing(0),
+    [theme.breakpoints.down("xs")]: {
+      fontSize: '0.875rem',
+    },
   },
   small: {
-    width: theme.spacing(3),
-    height: theme.spacing(3),
+    width: '32px',
```
Comment (IsAmrish) [inline website/src/components/DisplayAuthorandReadTime/styles.ts:35]: theme.spacing(4)

Comment (Pallavi-PH) [reply website/src/components/DisplayAuthorandReadTime/styles.ts:35]: theme spacing is generally used for padding and margin

File: website/src/pages/Blog/styles.ts [hunk 2]
```diff
@@ -3,12 +3,20 @@ import { makeStyles, Theme } from "@material-ui/core/styles";
 const useStyles = makeStyles((theme: Theme) => ({
   root: {
     position: "relative",
-    backgroundImage: `url(${"/assets/images/faq_background.png"})`,
+    backgroundImage: "url(/Images/png/faq_background.png)",
```
Comment (IsAmrish) [inline website/src/pages/Blog/styles.ts:6]: will have to change `Image` to `image` once @rakeshPRaghu 's PR (https://github.com/openebs/website/pull/121) will be merged

File: website/src/pages/Blog/styles.ts [hunk 3]
```diff
@@ -3,12 +3,20 @@ import { makeStyles, Theme } from "@material-ui/core/styles";
 const useStyles = makeStyles((theme: Theme) => ({
   root: {
     position: "relative",
-    backgroundImage: `url(${"/assets/images/faq_background.png"})`,
```
Comment (rakeshPRaghu) [inline website/src/pages/Blog/styles.ts:6]: we can remove this file!

File: website/src/pages/Blog/styles.ts (hunk 2 above)
Comment (Pallavi-PH) [reply website/src/pages/Blog/styles.ts:6]: Yes, once Rakesh's PR is merged, will make this change

File: website/src/components/DisplayAuthorandReadTime/styles.ts (hunk 1 above)
Comment (Pallavi-PH) [reply website/src/components/DisplayAuthorandReadTime/styles.ts:35]: https://material-ui.com/customization/spacing/

File: website/src/pages/CommercialSupport/styles.ts [hunk 4]
```diff
@@ -88,7 +88,8 @@ const useStyles = makeStyles((theme: Theme) => ({
     },
     [theme.breakpoints.down("xs")]: {
       boxShadow:' 0px 4px 34px 21px rgba(70, 68, 151, 0.04)',
-      borderRadius: '20px 20px 40px 0px'
+      borderRadius: '20px 20px 40px 0px',
```
Comment (IsAmrish) [inline website/src/pages/CommercialSupport/styles.ts:91]: where are we using theme spacing and where we are not?
kind of confused 😕

File: website/src/pages/Blog/styles.ts [hunk 5]
```diff
@@ -3,12 +3,20 @@ import { makeStyles, Theme } from "@material-ui/core/styles";
...
     backgroundPosition: "center top",
     backgroundSize: "cover",
     padding: theme.spacing(16, 5, 0),
+    [theme.breakpoints.down("xs")]: {
+      padding: theme.spacing(5, 5),
+      marginTop: theme.spacing(10),
+      backgroundImage: "url(/Images/png/faq_background_mobile.png)",
+      textAlign: "center",
+      backgroundRepeat: "no-repeat",
+      backgroundSize: "cover",
```
Comment (rakeshPRaghu) [inline website/src/pages/Blog/styles.ts:18]: same styles dont have to repeat

File: website/src/pages/CommercialSupport/styles.ts (hunk 4 above)
Comment (Pallavi-PH) [reply website/src/pages/CommercialSupport/styles.ts:91]: We are using theme spacing for margins and paddings only

File: website/src/components/DisplayAuthorandReadTime/styles.ts (hunk 1 above)
Comment (IsAmrish) [reply website/src/components/DisplayAuthorandReadTime/styles.ts:35]: got it @Pallavi-PH , thanks!!

File: website/src/pages/Blog/styles.ts (hunk 3 above)
Comment (Pallavi-PH) [reply website/src/pages/Blog/styles.ts:6]: Done

File: website/src/pages/Blog/styles.ts (hunk 5 above)
Comment (Pallavi-PH) [reply website/src/pages/Blog/styles.ts:17]: Removed `text-align`. Kept the rest of the styling. When the background image is changed using breakpoint, we need to provide other background image styling as well as that will not be inherited

**Review by AVRahul — APPROVED** [review]

**Review by IsAmrish — APPROVED** [review]

**Pull request merged**